providers/ruleset.rb

```diff
@@ -18,9 +18,12 @@
 #
 
 # provider for installing audit rules from a template
+
 action :create do
-  template '/etc/audit/audit.rules' do
+  t = template '/etc/audit/audit.rules' do
     source "#{new_resource.name}.erb"
-    notifies :restart, resources(service: 'auditd')
+    notifies :run, 'execute[auditctl -R]', :immediately
```

Comment:
Admittedly I am not yet overly familiar with auditd and was not aware of a push towards augenrules. It is a useful feature but I figured that Chef's flexibility negated the need for it. Feel free to add support for it and I will adjust this pull request accordingly but please do not throw false accusations around when my changes were correct for the cookbook in its current state. You may be trying to be funny but it isn't coming off very well.